### PR TITLE
Updated Little Snitch nightly to version 3.6-nightly-(4348)

### PR DESCRIPTION
**Pull request by bkono**
Rolled the version number forward, and updated the sha256.

### CODE REVIEW

File: Casks/little-snitch-nightly.rb [hunk 1]
```diff
@@ -1,7 +1,6 @@
 cask :v1 => 'little-snitch-nightly' do
-  version '3.4-nightly-(4208)'
-  sha256 'e3b91297945dcaaf7707f8f81301635012d56ea5342d817e21239d9b3f0945e7'
-
```
Comment (vitorgalvao) [inline Casks/little-snitch-nightly.rb]: The empty line is there for a reason. Please don’t remove it.

**Comment by bkono** [comment]
That was a whoops, thanks for catching. Double checked the nightly url hadn't changed, and accidentally killed the blank space. Its back.

**Comment by vitorgalvao** [comment]
Thank you. Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by bkono** [comment]
Happily. :)

**Pull request merged**